lib/mini_racer/truffleruby.rb

```diff
@@ -0,0 +1,304 @@
+# frozen_string_literal: true
+
+require "monitor"
+
+module MiniRacer
```

Comment:
Could you fix the indentation in this file to be 2 spaces?

Reply:
Fixed